boards/shields/amg88xx/Kconfig.shield

```diff
@@ -1,6 +1,9 @@
-# Copyright (c) 2022 Nordic Semiconductor ASA
+# Copyright (c) 2022 Panasonic Industrial Devices Europe GmbH
```

Comment:
You may not replace Copyright line, usually I  do not change it at all when a file is as simple as this one, git history is sufficient for that.